Core.ApplicationServices/Organizations/Write/OrganizationUnitWriteService.cs

```diff
@@ -87,6 +87,21 @@ public Result<OrganizationUnit, OperationError> Patch(Guid organizationUuid, Gui
             return result;
         }
 
+        public Maybe<OperationError> Delete(Guid organizationUuid, Guid organizationUnitUuid)
+        {
+            using var transaction = _transactionManager.Begin();
```

Comment:
I think you can straight up use the unit service you're calling here in the controller.

It already contains the transaction handling and uses uuids.

Reply:
So this one can be removed